src/index.ts

```diff
@@ -20,12 +20,15 @@ import TriggerEventType from './TriggerEventType';
 
 export class Popgun {
 
-  listener: EventListener = null;
-
   // registers mutation observer and sets up eventListeners
-  public constructor() {
-    mutationHandler.registerObserver();
-    eventDelegate.init();
+  public init(): void {
+    if (!document.body.hasAttribute('popgunExists')) {
+      mutationHandler.registerObserver();
+      eventDelegate.init();
+      document.body.setAttribute('popgunExists', '');
+    } else {
+      console.error('Popgun has already been instantiated. Do not instantiate again.');
```

Comment:
I don't remember if console.error has a stack trace associated or not, but I think I'd prefer to throw new Error('...') instead. Also because https://developer.mozilla.org/en-US/docs/Web/API/Console/error suggests that  this is a non-standard feature.